src/auth/epics/index.js

```diff
@@ -19,6 +20,19 @@ export function authEpicFactory() {
             }),
         );
 
+    const registerUserConfirm = action$ =>
+        action$.pipe(
+            ofType(actions.REGISTER_USER_CONFIRM),
+            switchMap(action =>
+                authApi
+                    .registerUserConfirm(action.payload)
+                    .catch(actions.registerUserConfirmRejected),
+            ),
+            map(res => {
+                return actions.registerUserConfirmFulfilled();
+            }),
+        );
```

Comment:
This won't work as expected. If registerUserConfirm will throw an error, the action `registerUserConfirmRejected` will be `mapped` to `registerUserConfirmFulfilled`. Basically: `registerUserConfirmRejected` will never run.
```
    const registerUserConfirm = action$ =>
        action$.pipe(
            ofType(actions.REGISTER_USER_CONFIRM),
            switchMap(action =>
                authApi
                    .registerUserConfirm(action.payload)
                    .then(actions.registerUserConfirmFulfilled)
                    .catch(actions.registerUserConfirmRejected),
            ),
        );
```

Reply:
yup, it's still not implemented, just copied. Will change then ;)